apps/client/src/app/(admin)/_components/upload-form/index.tsx

```
@@ -0,0 +1,49 @@
+      <Tags />
+
+      {/* File Uploads */}
+      <FileUploads />
+
+      {/* Submit */}
+      <div className="flex justify-end space-x-4">
+        <Button type="button" variant="outline">
+          Save as Draft
+        </Button>
+        <Button type="submit" disabled={false}>
+          {false ? (
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)